x-pack/legacy/plugins/maps/public/layers/styles/heatmap/heatmap_style.js

```diff
@@ -81,7 +81,7 @@ export class HeatmapStyle extends AbstractStyle {
 
     const { colorRampName } = this._descriptor;
     if (colorRampName && colorRampName !== DEFAULT_HEATMAP_COLOR_RAMP_NAME) {
-      const colorStops = getOrdinalColorRampStops(colorRampName);
+      const colorStops = getOrdinalColorRampStops(colorRampName, 0, 1);
```

Comment:
Would it be worth assigning `0` and `1` to named constants here? Might be hard to follow in the future